qutebrowser/misc/sql.py

```diff
@@ -318,19 +390,20 @@ def __len__(self):
         q.run()
         return q.value()
 
-    def delete(self, field, value, *, optional=False):
+    def delete(self, field, value, *, optional=False, like=False):
         """Remove all rows for which `field` equals `value`.
 
         Args:
             field: Field to use as the key.
             value: Key value to delete.
             optional: If set, non-existent values are ignored.
+            like: If set, LIKE is used instead of =
```

Comment:
doesn't look to be used anymore

Reply:
You're right - removed in fb5e105c1998350cd89512323c641be0d5d9d1b5.